Jenkinsfile

```diff
@@ -25,6 +25,12 @@ dockerNode(image: 'uf-mil:subjugator') {
 	}
 	stage("Format") {
 		sh '''
+			if [[ ! -z "$(python2.7 -m flake8 --max-line-length=120 --exclude=__init__.py .)" ]]; then
```

Comment:
I kinda like that, but at the same time, won't we start doing that with everything?

This rule isn't terribly restrictive.